content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Quantity
+id: Qty
+date: 2023-01-24
+full_link: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/
```

Comment:
There is no page about the quantity concept, so:
```suggestion
```

Reply:
do you prefer any other link or i should just remove this.